cmd/spire-server/cli/entry/util.go

```diff
@@ -47,6 +47,10 @@ func printEntry(e *types.Entry, env *common_cli.Env) {
 		env.Printf("TTL           : %d\n", e.Ttl)
 	}
 
+	if e.ExpiresAt != 0 {
+		env.Printf("ExpiresAt     : %d\n", e.ExpiresAt)
```

Comment:
Since the output is meant to be consumed by humans, I think that we should show this formatted as a readable date. That would also be consistent with other places where we show expiration time, like in the `agent list` command.

```suggestion
		env.Printf("Expires At    : %s\n", time.Unix(e.ExpiresAt, 0))
```

Reply:
To keep consistency, I'm thinking that maybe `Expiration time` is a more appropriate label?